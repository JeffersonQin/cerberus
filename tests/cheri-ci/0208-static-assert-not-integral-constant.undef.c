int main() {
  _Static_assert("a string", "is UB");
}
