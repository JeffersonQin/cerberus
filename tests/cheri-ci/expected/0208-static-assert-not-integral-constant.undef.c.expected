Undefined {ub: "UB204_illtyped_Static_assert", stderr: "", loc: "<2:18--2:28>"}
